packages/data-provider/src/utils/submittedDataUtils.ts

```diff
@@ -209,3 +253,21 @@ export const transformmSubmittedDataToSubmissionDeleteData = (submittedData: Sub
 		return acc;
 	}, {});
 };
+
+export const updateSubmittedDataArray = (
+	submittedData: SubmittedData[],
+	editData: SubmissionUpdateData[],
+): SubmittedData[] => {
+	return submittedData.map((existingSubmittedData) => {
+		const found = editData.find((e) => e.systemId == existingSubmittedData.systemId);
```

Comment:
the rare `==` instead of `===`, is there a reason?

Reply:
Thanks for noticing. Changed to `===` as both should be same string type